evm_arithmetization/src/generation/mod.rs

```diff
@@ -43,6 +43,9 @@ use crate::witness::util::mem_write_log;
 /// Number of cycles to go after having reached the halting state. It is
 /// equal to the number of cycles in `exc_stop` + 1.
 pub const NUM_EXTRA_CYCLES_AFTER: usize = 81;
+/// Number of cycles to go before starting the execution: it is the number of
+/// cycles in `init`.
+pub const NUM_EXTRA_CYCLES_BEFORE: usize = 64;
```

Comment:
random question, but do we have actual unit tests enforcing this?
Any change in the kernel optimization procedure may mess these numbers. We should catch this asap

Reply:
There was an assert to check `NUM_EXTRA_CYCLES_AFTER`, but I changed that into a `debug_assert` and added a unit test in `init_exc_stop.rs` to check both values.